Added websocket initPayload into the (server) context (#1818)
smallrye/smallrye-graphql: docs/handling-init-payload-from-the-websocket.md

Handling of the WebSocket's init-payload
===========

> When using GraphQLTransportWS or GraphQLWS, your client can send an initial payload

Reviewer: ```suggestion
When executing GraphQL operations over WebSockets (both with the `graphql-ws` and `graphql-transport-ws` protocols), the client can send an initial payload
```

to the server when establishing the WebSocket connection. This payload can contain
any information that the server might need to process GraphQL requests, like User-Agent header for HTTP
or data specific to the client. On the server-side, you can access this payload

> using the CDI `SmallRyeContext` `get` method of the `GraphQlContext` object, which is available through the

Reviewer: ```suggestion
via the `GraphQLContext`, which is available through the
```
